src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt

```diff
@@ -51,7 +52,7 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
       "where u.prison.code = :prisonCode " +
```

Comment:
`findActiveAndFutureSessionTemplates`
should be
`findCurrentAndFutureSessionTemplates`

Reply:
Updated